src/app/modules/item/components/thread-container/thread-container.component.scss

```diff
@@ -0,0 +1,7 @@
+.__container {
```

Comment:
what's the semantic of the double underscore?

Reply:
This is BEM methodology, my old habit, so i've added it by mistake. Hope in future we'll use some methodology as well.

Fixed